codegenerator/cli/templates/static/codegen/src/bindings/HyperFuelClient.res

```diff
@@ -0,0 +1,497 @@
+type unchecksummedEthAddress = string
+
+type t
+
+type cfg = {
+  url: string,
+  bearerToken?: string,
+  http_req_timeout_millis?: int,
+}
+module QueryTypes = {
+  type blockFieldOptions =
+    | @as("id") Id
+    | @as("da_height") DaHeight
+    | @as("transactions_count") TransactionsCount
+    | @as("message_receipt_count") MessageReceiptCount
+    | @as("transactions_root") TransactionsRoot
+    | @as("message_receipt_root") MessageReceiptRoot
```

Comment:
I'm keen at some point to add an exhaustive binding in the actual client package.